# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../vdr/vcd-0.7-1.3.38.diff
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -ruN vcd-0.7-original/menucontrol.c vcd-0.7/menucontrol.c
--- vcd-0.7-original/menucontrol.c	2005-01-28 16:06:48.000000000 +0100
+++ vcd-0.7/menucontrol.c	2006-01-09 13:13:35.393131232 +0100
@@ -39,7 +39,7 @@
 
 cMenuSpiControl::~cMenuSpiControl()
 {
-  cStatus::MsgReplaying(this, NULL);
+  cStatus::MsgReplaying(this, NULL, NULL, true);
   Stop();
 }
 
@@ -118,13 +118,13 @@
   lastSpeed = -1;
   timeoutShow = 0;
   timeSearchActive = false;
-  cStatus::MsgReplaying(this, "VCD");
+  cStatus::MsgReplaying(this, "VCD", NULL, true);
 }
 
 cMenuVcdControl::~cMenuVcdControl()
 {
   Hide();
-  cStatus::MsgReplaying(this, NULL);
+  cStatus::MsgReplaying(this, NULL, NULL, false);
   Stop();
 }
 
diff -ruN vcd-0.7-original/psdcontrol.c vcd-0.7/psdcontrol.c
--- vcd-0.7-original/psdcontrol.c	2005-01-28 20:15:14.000000000 +0100
+++ vcd-0.7/psdcontrol.c	2006-01-09 13:15:43.907594064 +0100
@@ -84,7 +84,7 @@
  
 cPsdSpiControl::~cPsdSpiControl()
 {
-  cStatus::MsgReplaying(this, NULL);
+  cStatus::MsgReplaying(this, NULL, NULL, true);
 }
 
 void cPsdSpiControl::SetItem(int Spi, cVcd *Vcd, union psd_vcd *PsdVcd)
@@ -252,7 +252,7 @@
 
 cPsdVcdControl::~cPsdVcdControl()
 {
-  cStatus::MsgReplaying(this, NULL);
+  cStatus::MsgReplaying(this, NULL, NULL, true);
 }
 
 void cPsdVcdControl::SetTrack(int Track, cVcd *Vcd, union psd_vcd *PsdVcd)
